### DISCUSSION TITLE
workflow: how can I send a email for all users based on a rule

### DISCUSSION THREAD

**Discussion by AmadeuAntunes** [OrchardCMS/OrchardCore#8281] [Q&A]
I'm trying to create a workflow that send a email for all users that contain some rule

**Comment by hishamco** [comment]
If you want to send an email to all the users registered, I think you need to extend `EmailTask`  to get the user's emails then send the email to them

**Comment by Skrypt** [comment]
I strongly suggest using a mass mailing service to do this. Workflows email task is not optimized for this.
